hs_tools_resource/models.py

```diff
@@ -597,6 +621,11 @@ def parse_for_bulk_update(cls, metadata, parsed_metadata):
         """Overriding the base class method"""
 
         CoreMetaData.parse_for_bulk_update(metadata, parsed_metadata)
+        # TODO The json metadata has underscores, remove those to matche the metadata elements.  This is probably an
+        # issue in the other metadata implementations as well.
+        for key, value in metadata.items():
+            if "_" in key:
+                metadata[key.replace("_", "")] = metadata.pop(key)
```

Comment:
do not quite understand why we don't keep JSON and metadata consistent, either both have underscores or not, then this block would not be needed.

Reply:
I don't get it either, this is the best solution I could come up with to make it more usable without breaking anything that may be dependent on the no underscores metadata names.